docs/spec/spec.md

```diff
@@ -135,6 +135,8 @@ import ballerina/graphql;
 ### 2.1 Listener
 
 Since the GraphQL spec does not mandate an underlying client-server protocol, a GraphQL implementation can use any protocol underneath. The Ballerina GraphQL package, like most of the other implementations, uses HTTP as the protocol. The Ballerina GraphQL listener is using an HTTP listener to listen to incoming requests through HTTP.
+If the schema contains the `Subscription` type, The GraphQL listener will establish a new WebSocket listener to listen to incoming subscription requests.
```

Comment:
Let's add two sub-topics for `2.1.1 HTTP Listener` and `2.1.2 WebSocket Listener`.